lib/kaiser/cmds/down.rb

```diff
@@ -12,7 +12,9 @@ def usage
       end
 
       def execute(_opts)
-        down
+        stop_db
+        stop_app
+        delete_db_volume
```

Comment:
This is a cleanup. There used to be a `#down` but there doesnt have to be anymore